plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineStandardWizardPage.java

```diff
@@ -67,9 +80,31 @@ public void createControl(Composite parent) {
     projectIdField.setLayoutData(projectIdPosition);
     projectIdField.addModifyListener(pageValidator);
 
+    // Manage APIs
+    addManageLibrariesWidgets(container);
+
     Dialog.applyDialogFont(container);
   }
 
+  private void addManageLibrariesWidgets(Composite container) {
+    apiGroup = new Group(container, SWT.NONE);
+    apiGroup.setText("Manage Libraries");
```

Comment:
I could be wrong, but it feels to me like a noun is needed in this label, not a verb because clicking the checkbox does not, itself, take an action.